### PR TITLE
Upgrae to v3.59.0 of the Google Beta Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #536

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -918,6 +920,7 @@ func Provider() tfbridge.ProviderInfo {
 				},
 			},
 			"google_dataproc_autoscaling_policy": {Tok: gcpResource(gcpDataProc, "AutoscalingPolicy")},
+			"google_dataproc_metastore_service":  {Tok: gcpResource(gcpDataProc, "MetaStoreService")},
```
Comment (mikhailshilkov) [inline provider/resources.go:923]: It seems to be "Metastore" in the docs https://cloud.google.com/dataproc-metastore/docs

File: sdk/dotnet/Redis/Instance.cs [hunk 2]
```diff
@@ -330,12 +330,6 @@ public sealed class InstanceArgs : Pulumi.ResourceArgs
         [Input("authEnabled")]
         public Input<bool>? AuthEnabled { get; set; }
 
-        /// <summary>
-        /// AUTH String set on the instance. This field will only be populated if auth_enabled is true.
-        /// </summary>
-        [Input("authString")]
-        public Input<string>? AuthString { get; set; }
```
Comment (mikhailshilkov) [inline sdk/dotnet/Redis/Instance.cs:337]: Breaking change?

Comment (stack72) [reply sdk/dotnet/Redis/Instance.cs:337]: This is expected - this was never a valid input - it's a computed field only

File: provider/resources.go (hunk 1 above)
Comment (stack72) [reply provider/resources.go]: Fixed!

**Review by mikhailshilkov — APPROVED** [review]

**Pull request merged**